### PR TITLE
docs: update name jsdocs

### PR DESCRIPTION
**Pull request by ST-DDT**

### CODE REVIEW

**Review by Shinigami92 — APPROVED** [review]

File: src/name.ts [hunk 1]
```diff
@@ -187,11 +214,13 @@ export class Name {
   }
 
   /**
-   * gender
+   * Return a random gender name.
```
Comment (griest024) [inline src/name.ts:217]: It can just be `Return a random gender.`

File: src/name.ts [hunk 2]
```diff
@@ -187,11 +214,13 @@ export class Name {
...
+   *
+   * @example
+   * faker.name.jobTitle() // 'Trans*Man'
+   * faker.name.jobTitle(true) // 'Female'
```
Comment (griest024) [inline src/name.ts:221]: Example methods are wrong

**Review by griest024 — CHANGES_REQUESTED** [review]

File: src/name.ts (hunk 1 above)
Comment (ST-DDT) [reply src/name.ts]: Fixed

File: src/name.ts (hunk 2 above)
Comment (ST-DDT) [reply src/name.ts]: Fixed

**Review by Shinigami92 — APPROVED** [review]

**Review by griest024 — APPROVED** [review]

**Pull request merged**